aiida/tests/work/test_daemon.py

```diff
@@ -1,156 +0,0 @@
-
```

Comment:
This was a mistake I think by Sebastiaan which somehow hung around.  It is now moved into the backends test section: aiida/backends/tests/work/daemon otherwise it would try to load_dbenv and fail.